vnext/ReactUWP/Modules/NativeUIManager.h

```diff
@@ -53,6 +53,12 @@ class NativeUIManager : public facebook::react::INativeUIManager
   // Other public functions
   void DirtyYogaNode(int64_t tag);
 
+  // For unparented node like Flyout, XamlRoot should be set to handle XamlIsland/AppWindow scenarios.
+  // Since it doesn't have parent, and all nodes in the tree should have the same XamlRoot,
+  // this function iterates all roots and try to get a valid XamlRoot.
+  // To reduce the dependency to cppwinrt in header file, ShadowNode* other XamlRoot is returned.
```

Comment:
>ShadowNode* other XamlRoot is returned [](start = 58, length = 38)

Can you forward declare XamlRoot to solve this?

Reply:
I just changed it to winrt::XamlRoot.